spec/daru/io/exporters/csv_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe Daru::IO::Exporters::CSV do
+  include_context 'exporter setup'
+  let(:filename) { 'test.csv' }
+  subject        { File.open(tempfile.path, &:readline).chomp.split(',', -1) }
+
+  before { Daru::IO::Exporters::CSV.new(df, tempfile.path, opts).write }
```

Comment:
Note to self : Change this `Daru::IO::Exporters::CSV.new(df, tempfile.path, opts).write` to `described_class.new(df, tempfile.path, opts).write` here and similarly in other spec files.